schemas/post.py

```
@@ -0,0 +1,65 @@
+    title: Title
+    content: Content
+
+
+class PostUpdateRequest(BaseModel):
+    title: Title | None = None
+    content: Content | None = None
+
+    @model_validator(mode='after')
+    def at_least_one_field(self):
+        if self.title is None and self.content is None:
+            raise ValueError("수정할 필드가 없습니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)